configure.ac

```diff
@@ -412,6 +412,8 @@ COOLWSD_LOG_TO_FILE="false"
 COOLWSD_LOGFILE="/var/log/coolwsd.log"
 COOLWSD_TRACEEVENTFILE="/var/log/coolwsd.trace.json"
 COOLWSD_ANONYMIZE_USER_DATA=false
+COOLWSD_LOG_UICMD_TO_FILE="false"
```

Comment:
line 479 of COOLWSD_LOG_TO_FILE_UICMD doesn't match here, this 415 line should likely be fixed to match the other

Reply:
Thx, i fixed the name problem :)